graphql_compiler/compiler/emit_sql.py

```diff
@@ -143,21 +143,34 @@ def _find_used_columns(
             )
             vertex_field_name = f"{edge_direction}_{edge_name}"
             edge = sql_schema_info.join_descriptors[location_info.type.name][vertex_field_name]
-            used_columns.setdefault(get_vertex_path(location), set()).add(edge.from_column)
-            used_columns.setdefault(get_vertex_path(child_location), set()).add(edge.to_column)
+            if isinstance(edge, DirectJoinDescriptor):
+                columns_at_location = {edge.from_column}
+                columns_at_child = {edge.to_column}
+            elif isinstance(edge, CompositeJoinDescriptor):
+                columns_at_location = {
+                    column_pair[0]
+                    for column_pair in edge.column_pairs
+                }
+                columns_at_child = {
+                    column_pair[1]
+                    for column_pair in edge.column_pairs
+                }
```

Comment:
Is the `else` here a `pass` or a `raise AssertionError` kind of `else`? Not sure what the inferred type of `edge` is on line 145, but hoping we can make that explicit.

Reply:
Yes! Ty for catching it.